Added basic auth to webpage

sambelltacq opened the pull request:
new file:   etc/nginx/nginx_ssl_forced_auth.conf
	modified:   init/httpd.init

sambelltacq committed:
Added basic auth to webpage
	new file:   etc/nginx/nginx_ssl_forced_auth.conf
	modified:   init/httpd.init

petermilne commented:
Sam, I like your new feature very much. A small change request:

DONT replace this
non_volatile=/mnt/local/sysconfig/ssl
nv_cert=${non_volatile}/$(hostname).crt
nv_key=${non_volatile}/$(hostname).key
with this:
sysconfig=/mnt/local/sysconfig/

Sure $sysconfig says "where", it doesn't give us any more info than "/mnt/local/sysconfig/" (also "where") but
$non_volatile says "why" which is more important, and also explains what nv_xxx means.

sambelltacq committed:
Improved nginx logging and fixed auth name
    modified:   etc/nginx/nginx.conf
	modified:   etc/nginx/nginx_ssl.conf
	modified:   etc/nginx/nginx_ssl_forced.conf
	modified:   etc/nginx/nginx_ssl_forced_auth.conf

sambelltacq committed:
Renamed variable
	modified:   init/httpd.init
